src/otx/engine/engine.py

```diff
@@ -276,19 +276,24 @@ def test(
                 otx test --config <CONFIG_PATH, str> --checkpoint <CKPT_PATH, str>
                 ```
         """
+        model = self.model
+        checkpoint = checkpoint if checkpoint is not None else self.checkpoint
+        datamodule = datamodule if datamodule is not None else self.datamodule
+
+        is_ir_ckpt = str(checkpoint).endswith(".xml")
```

Comment:
```suggestion
        is_ir_ckpt = checkpoint.suffix == "xml"
```
Minor one, but `Path` can give suffix.

Reply:
Checkpoint can be also an onnx file (MAPI allows that)

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/2897/commits/d8ab60c65a179e36732287ee3d4fec5e8e7bf0e5